### ISSUE TITLE
Unresponsive on FTP server drop out

### ISSUE THREAD

**Issue by AliveDevil** [iterate-ch/cyberduck#15671]
When a download transfer is in progress (multiple connections, segmented) the moment the FTP server fails (IP available, Port dead) the macOS app becomes unresponsive.

[Spindump.txt](https://github.com/iterate-ch/cyberduck/files/14447874/Spindump.txt)

**Comment by AliveDevil** [comment]
[threaddump-1709213262090.tdump.txt](https://github.com/iterate-ch/cyberduck/files/14448622/threaddump-1709213262090.tdump.txt)

**Comment by dkocher** [comment]
The thread dump shows that the `main` thread is waiting for all transfer threads to complete. 

```
	at ch.cyberduck.core.threading.ExecutorServiceThreadPool.shutdown(ExecutorServiceThreadPool.java:50)
	at ch.cyberduck.core.worker.ConcurrentTransferWorker.shutdown(ConcurrentTransferWorker.java:183)
	at ch.cyberduck.core.worker.AbstractTransferWorker.cleanup(AbstractTransferWorker.java:548)
	at ch.cyberduck.core.worker.AbstractTransferWorker.cleanup(AbstractTransferWorker.java:53)
	at ch.cyberduck.core.threading.TransferWorkerBackgroundAction.cleanup(TransferWorkerBackgroundAction.java:67)
	at ch.cyberduck.core.threading.TransferCollectionBackgroundAction.cleanup(TransferCollectionBackgroundAction.java:90)
	at ch.cyberduck.ui.cocoa.controller.TransferController$8.cleanup(TransferController.java:624)
```

Regression from 238ac9c8.

**Comment by dkocher** [comment]
Was previously fixed in fa618e56.
